payment-intent/cs/payment-intent-auto.cs

```diff
@@ -0,0 +1,16 @@
+var paymentIntentsClient = new PaymentIntentsClient(new HttpClient(), new ApiKeyClientAuthorization("sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ"));
```

Comment:
I'm not sure that you need this sample at all, because this is default one, and there is already sample how to create payment intent

Reply:
To maintain consistency, we have added again as we have tabs for the languages. Any thoughts?

Reply:
Updated existing create payment intent documentation with example added. And removed the other instance in auto mode. Will now add link to the existing file instead.